.eslintrc.js

```diff
@@ -1,6 +1,9 @@
 module.exports = {
   // parser: 'babel-eslint',
   parser: '@typescript-eslint/parser',
+  env: {
```

Comment:
Nit: I'd prefer an explicit `/* eslint-env browser */` comment on files where DOM APIs are used; that will allow ESLint to warn us if, for instance, someone accidentally tries to use a browser API in server code.